context_prop/src/main/java/io/opentelemetry/context/GlobalContextPropagators.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.context;
+
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.context.propagation.DefaultContextPropagators;
+
+/**
+ * This class provides access to for a global `ContextPropagators` object.
+ *
+ * <p>The default value is a no-op instance.
+ */
+public final class GlobalContextPropagators {
```

Comment:
I think I'd rather not have the `Global` prefix on this class. What do you think?

Reply:
Sure, but I couldn't come with a prefix I'd feel comfortable with. `ContextPropagators` is already defined under the `propagation` sub-package. But I'm open to other options definitely.